Dockerfile

```diff
@@ -5,6 +5,14 @@ WORKDIR /phoenixapp
 COPY ./ /phoenixapp
 RUN mv  /transport-tools /phoenixapp
 
+# Installation of AppSignal dependencies:
+# https://docs.appsignal.com/support/operating-systems.html#debian-ubuntu
+# To be moved to the base Docker image instead:
+# https://github.com/etalab/transport-ops/issues/51
+# (but this could take a bit of time)
```

Comment:
Je me demande si l'installation des dépendances doit être bien récent ou pas, l'image du dessus n'est pas rafraichie très régulièrement non plus

Reply:
Oui c'est vrai... J'ai regardé de plus près, au final ça prend 35 secondes de plus, et surtout ça met à jour d'autres choses en pagaille.

Ca pourrait casser quelque chose plus facilement. L'idéal serait que l'image de base soit mise à jour plus régulièrement mais avec des tests.

On en reparle ?

```
2023-07-04T09:20:32+02:00 Step 6/18 : RUN apt-get update
2023-07-04T09:21:08+02:00 # SNIP
```

<details>
<summary>Click</summary>

```
2023-07-04T09:20:32+02:00 Step 6/18 : RUN apt-get update
2023-07-04T09:20:32+02:00 ---> Running in bef5c6ec07b0
2023-07-04T09:20:32+02:00 time="2023-07-04T07:20:32.207303935Z" level=info msg="loading plugin \"io.containerd.internal.v1.shutdown\"..." runtime=io.containerd.runc.v2 type=io.containerd.internal.v1
2023-07-04T09:20:32+02:00 time="2023-07-04T07:20:32.207455759Z" level=info msg="loading plugin \"io.containerd.ttrpc.v1.pause\"..." runtime=io.containerd.runc.v2 type=io.containerd.ttrpc.v1
2023-07-04T09:20:32+02:00 time="2023-07-04T07:20:32.207532166Z" level=info msg="loading plugin \"io.containerd.ttrpc.v1.task\"..." runtime=io.containerd.runc.v2 type=io.containerd.ttrpc.v1
2023-07-04T09:20:32+02:00 time="2023-07-04T07:20:32.207495417Z" level=info msg="loading plugin \"io.containerd.event.v1.publisher\"..." runtime=io.containerd.runc.v2 type=io.containerd.event.v1
2023-07-04T09:20:32+02:00 Hit:1 http://archive.ubuntu.com/ubuntu focal InRelease
2023-07-04T09:20:32+02:00 Get:3 http://archive.ubuntu.com/ubuntu focal-backports InRelease [108 kB]
2023-07-04T09:20:32+02:00 Get:2 http://archive.ubuntu.com/ubuntu focal-updates InRelease [114 kB]
2023-07-04T09:20:32+02:00 Get:4 http://security.ubuntu.com/ubuntu focal-security InRelease [114 kB]
2023-07-04T09:20:33+02:00 Get:5 http://archive.ubuntu.com/ubuntu focal-updates/multiverse amd64 Packages [31.2 kB]
2023-07-04T09:20:33+02:00 Get:7 http://archive.ubuntu.com/ubuntu focal-updates/restricted amd64 Packages [2642 kB]
2023-07-04T09:20:33+02:00 Get:6 http://archive.ubuntu.com/ubuntu focal-updates/universe amd64 Packages [1366 kB]
2023-07-04T09:20:33+02:00 Get:8 http://archive.ubuntu.com/ubuntu focal-updates/main amd64 Packages [3338 kB]
2023-07-04T09:20:33+02:00 Get:10 http://archive.ubuntu.com/ubuntu focal-backports/main amd64 Packages [55.2 kB]
2023-07-04T09:20:33+02:00 Get:9 http://archive.ubuntu.com/ubuntu focal-backports/universe amd64 Packages [28.6 kB]
2023-07-04T09:20:33+02:00 Get:11 http://security.ubuntu.com/ubuntu focal-security/main amd64 Packages [2858 kB]
2023-07-04T09:20:34+02:00 Get:14 http://security.ubuntu.com/ubuntu focal-security/restricted amd64 Packages [2479 kB]
2023-07-04T09:20:34+02:00 Get:13 http://security.ubuntu.com/ubuntu focal-security/multiverse amd64 Packages [28.5 kB]
2023-07-04T09:20:34+02:00 Get:12 http://security.ubuntu.com/ubuntu focal-security/universe amd64 Packages [1066 kB]
2023-07-04T09:20:34+02:00 Fetched 14.2 MB in 2s (6948 kB/s)
2023-07-04T09:20:35+02:00 Reading package lists...
2023-07-04T09:20:35+02:00 time="2023-07-04T07:20:35.720876656Z" level=info msg="shim disconnected" id=bef5c6ec07b0a0b64394e55ef30910785590863b2c20230a252ff59cc0215541 namespace=moby
2023-07-04T09:20:35+02:00 time="2023-07-04T07:20:35.720916827Z" level=info msg="ignoring event" container=bef5c6ec07b0a0b64394e55ef30910785590863b2c20230a252ff59cc0215541 module=libcontainerd namespace=moby topic=/tasks/delete type="*events.TaskDelete"
2023-07-04T09:20:35+02:00 time="2023-07-04T07:20:35.720955159Z" level=info msg="cleaning up dead shim" namespace=moby
2023-07-04T09:20:35+02:00 time="2023-07-04T07:20:35.720943039Z" level=warning msg="cleaning up after shim disconnected" id=bef5c6ec07b0a0b64394e55ef30910785590863b2c20230a252ff59cc0215541 namespace=moby
2023-07-04T09:20:37+02:00 ---> Running in 44f799d3020e
2023-07-04T09:20:37+02:00 Removing intermediate container bef5c6ec07b0
2023-07-04T09:20:37+02:00 Step 7/18 : RUN apt-get install --assume-yes build-essential ca-certificates
2023-07-04T09:20:37+02:00 time="2023-07-04T07:20:37.823629472Z" level=info msg="loading plugin \"io.containerd.event.v1.publisher\"..." runtime=io.containerd.runc.v2 type=io.containerd.event.v1
2023-07-04T09:20:37+02:00 time="2023-07-04T07:20:37.823596908Z" level=info msg="loading plugin \"io.containerd.ttrpc.v1.pause\"..." runtime=io.containerd.runc.v2 type=io.containerd.ttrpc.v1
2023-07-04T09:20:37+02:00 time="2023-07-04T07:20:37.823444500Z" level=info msg="loading plugin \"io.containerd.internal.v1.shutdown\"..." runtime=io.containerd.runc.v2 type=io.containerd.internal.v1
2023-07-04T09:20:37+02:00 time="2023-07-04T07:20:37.823650145Z" level=info msg="loading plugin \"io.containerd.ttrpc.v1.task\"..." runtime=io.containerd.runc.v2 type=io.containerd.ttrpc.v1
2023-07-04T09:20:37+02:00 ---> 10343ca8d4be
2023-07-04T09:20:39+02:00 Reading state information...
2023-07-04T09:20:39+02:00 Building dependency tree...
2023-07-04T09:20:39+02:00 Reading package lists...
2023-07-04T09:20:39+02:00 dirmngr dpkg-dev fakeroot g++ g++-9 gnupg gnupg-l10n gnupg-utils gpg
2023-07-04T09:20:39+02:00 The following additional packages will be installed:
2023-07-04T09:20:39+02:00 libfakeroot libfile-fcntllock-perl libksba8 liblocale-gettext-perl libnpth0
2023-07-04T09:20:39+02:00 libalgorithm-diff-xs-perl libalgorithm-merge-perl libassuan0 libdpkg-perl
2023-07-04T09:20:39+02:00 gpg-agent gpg-wks-client gpg-wks-server gpgconf gpgsm libalgorithm-diff-perl
2023-07-04T09:20:39+02:00 libstdc++-9-doc make-doc pinentry-doc readline-doc
2023-07-04T09:20:39+02:00 dbus-user-session libpam-systemd pinentry-gnome3 tor debian-keyring
2023-07-04T09:20:39+02:00 Suggested packages:
2023-07-04T09:20:39+02:00 libreadline8 libstdc++-9-dev make pinentry-curses readline-common xz-utils
2023-07-04T09:20:39+02:00 g++-multilib g++-9-multilib gcc-9-doc parcimonie xloadimage scdaemon bzr
2023-07-04T09:20:39+02:00 libalgorithm-diff-perl libalgorithm-diff-xs-perl libalgorithm-merge-perl
2023-07-04T09:20:39+02:00 gnupg-utils gpg gpg-agent gpg-wks-client gpg-wks-server gpgconf gpgsm
2023-07-04T09:20:39+02:00 libassuan0 libdpkg-perl libfakeroot libfile-fcntllock-perl libksba8
2023-07-04T09:20:39+02:00 build-essential dirmngr dpkg-dev fakeroot g++ g++-9 gnupg gnupg-l10n
2023-07-04T09:20:39+02:00 liblocale-gettext-perl libnpth0 libreadline8 libstdc++-9-dev make
2023-07-04T09:20:39+02:00 The following NEW packages will be installed:
2023-07-04T09:20:39+02:00 pinentry-curses readline-common xz-utils
2023-07-04T09:20:39+02:00 ca-certificates
2023-07-04T09:20:39+02:00 The following packages will be upgraded:
2023-07-04T09:20:46+02:00 After this operation, 60.7 MB of additional disk space will be used.
2023-07-04T09:20:46+02:00 Need to get 14.4 MB of archives.
2023-07-04T09:20:46+02:00 1 upgraded, 31 newly installed, 0 to remove and 50 not upgraded.
2023-07-04T09:20:46+02:00 Get:1 http://archive.ubuntu.com/ubuntu focal/main amd64 liblocale-gettext-perl amd64 1.07-4 [17.1 kB]
2023-07-04T09:20:46+02:00 Get:2 http://archive.ubuntu.com/ubuntu focal-updates/main amd64 ca-certificates all 20230311ubuntu0.20.04.1 [152 kB]
2023-07-04T09:20:47+02:00 Get:4 http://archive.ubuntu.com/ubuntu focal/main amd64 libreadline8 amd64 8.0-4 [131 kB]
2023-07-04T09:20:47+02:00 Get:5 http://archive.ubuntu.com/ubuntu focal-updates/main amd64 xz-utils amd64 5.2.4-1ubuntu1.1 [82.6 kB]
2023-07-04T09:20:47+02:00 Get:3 http://archive.ubuntu.com/ubuntu focal/main amd64 readline-common all 8.0-4 [53.5 kB]
2023-07-04T09:20:47+02:00 Get:6 http://archive.ubuntu.com/ubuntu focal-updates/main amd64 libstdc++-9-dev amd64 9.4.0-1ubuntu1~20.04.1 [1722 kB]
2023-07-04T09:20:47+02:00 Get:7 http://archive.ubuntu.com/ubuntu focal-updates/main amd64 g++-9 amd64 9.4.0-1ubuntu1~20.04.1 [8420 kB]
2023-07-04T09:20:47+02:00 Get:8 http://archive.ubuntu.com/ubuntu focal/main amd64 g++ amd64 4:9.3.0-1ubuntu2 [1604 B]
2023-07-04T09:20:47+02:00 Get:9 http://archive.ubuntu.com/ubuntu focal/main amd64 make amd64 4.2.1-1.2 [162 kB]
2023-07-04T09:20:47+02:00 Get:13 http://archive.ubuntu.com/ubuntu focal/main amd64 libassuan0 amd64 2.5.3-7ubuntu2 [35.7 kB]
2023-07-04T09:20:47+02:00 Get:12 http://archive.ubuntu.com/ubuntu focal-updates/main amd64 build-essential amd64 12.8ubuntu1.1 [4664 B]
2023-07-04T09:20:47+02:00 Get:11 http://archive.ubuntu.com/ubuntu focal-updates/main amd64 dpkg-dev all 1.19.7ubuntu3.2 [679 kB]
2023-07-04T09:20:47+02:00 Get:10 http://archive.ubuntu.com/ubuntu focal-updates/main amd64 libdpkg-perl all 1.19.7ubuntu3.2 [231 kB]
2023-07-04T09:20:48+02:00 Get:15 http://archive.ubuntu.com/ubuntu focal-updates/main amd64 libksba8 amd64 1.3.5-2ubuntu0.20.04.2 [95.2 kB]
2023-07-04T09:20:48+02:00 Get:14 http://archive.ubuntu.com/ubuntu focal-updates/main amd64 gpgconf amd64 2.2.19-3ubuntu2.2 [124 kB]
2023-07-04T09:20:48+02:00 Get:16 http://archive.ubuntu.com/ubuntu focal/main amd64 libnpth0 amd64 1.6-1 [7736 B]
2023-07-04T09:20:48+02:00 Get:19 http://archive.ubuntu.com/ubuntu focal/main amd64 fakeroot amd64 1.24-1 [62.6 kB]
2023-07-04T09:20:48+02:00 Get:20 http://archive.ubuntu.com/ubuntu focal-updates/main amd64 gnupg-l10n all 2.2.19-3ubuntu2.2 [51.7 kB]
2023-07-04T09:20:48+02:00 Get:18 http://archive.ubuntu.com/ubuntu focal/main amd64 libfakeroot amd64 1.24-1 [25.7 kB]
2023-07-04T09:20:48+02:00 Get:17 http://archive.ubuntu.com/ubuntu focal-updates/main amd64 dirmngr amd64 2.2.19-3ubuntu2.2 [330 kB]
2023-07-04T09:20:48+02:00 Get:21 http://archive.ubuntu.com/ubuntu focal-updates/main amd64 gnupg-utils amd64 2.2.19-3ubuntu2.2 [481 kB]
2023-07-04T09:20:48+02:00 Get:22 http://archive.ubuntu.com/ubuntu focal-updates/main amd64 gpg amd64 2.2.19-3ubuntu2.2 [482 kB]
2023-07-04T09:20:48+02:00 Get:23 http://archive.ubuntu.com/ubuntu focal/main amd64 pinentry-curses amd64 1.1.0-3build1 [36.3 kB]
2023-07-04T09:20:49+02:00 Get:25 http://archive.ubuntu.com/ubuntu focal-updates/main amd64 gpg-wks-client amd64 2.2.19-3ubuntu2.2 [97.4 kB]
2023-07-04T09:20:49+02:00 Get:24 http://archive.ubuntu.com/ubuntu focal-updates/main amd64 gpg-agent amd64 2.2.19-3ubuntu2.2 [232 kB]
2023-07-04T09:20:49+02:00 Get:26 http://archive.ubuntu.com/ubuntu focal-updates/main amd64 gpg-wks-server amd64 2.2.19-3ubuntu2.2 [90.2 kB]
2023-07-04T09:20:49+02:00 Get:31 http://archive.ubuntu.com/ubuntu focal/main amd64 libalgorithm-merge-perl all 0.08-3 [12.0 kB]
2023-07-04T09:20:49+02:00 Get:27 http://archive.ubuntu.com/ubuntu focal-updates/main amd64 gpgsm amd64 2.2.19-3ubuntu2.2 [217 kB]
2023-07-04T09:20:49+02:00 Get:30 http://archive.ubuntu.com/ubuntu focal/main amd64 libalgorithm-diff-xs-perl amd64 0.04-6 [11.3 kB]
2023-07-04T09:20:49+02:00 Get:32 http://archive.ubuntu.com/ubuntu focal/main amd64 libfile-fcntllock-perl amd64 0.22-3build4 [33.1 kB]
2023-07-04T09:20:49+02:00 Get:29 http://archive.ubuntu.com/ubuntu focal/main amd64 libalgorithm-diff-perl all 1.19.03-2 [46.6 kB]
2023-07-04T09:20:49+02:00 Get:28 http://archive.ubuntu.com/ubuntu focal-updates/main amd64 gnupg all 2.2.19-3ubuntu2.2 [259 kB]
2023-07-04T09:20:49+02:00 Selecting previously unselected package liblocale-gettext-perl.
2023-07-04T09:20:49+02:00 debconf: delaying package configuration, since apt-utils is not installed
2023-07-04T09:20:49+02:00 Preparing to unpack .../01-ca-certificates_20230311ubuntu0.20.04.1_all.deb ...
2023-07-04T09:20:49+02:00 Fetched 14.4 MB in 10s (1510 kB/s)
2023-07-04T09:20:49+02:00 (Reading database ... #015(Reading database ... 5%#015(Reading database ... 10%#015(Reading database ... 15%#015(Reading database ... 20%#015(Reading database ... 25%#015(Reading database ... 30%#015(Reading database ... 35%#015(Reading database ... 40%#015(Reading database ... 45%#015(Reading database ... 50%#015(Reading database ... 55%#015(Reading database ... 60%#015(Reading database ... 65%#015(Reading database ... 70%#015(Reading database ... 75%#015(Reading database ... 80%#015(Reading database ... 85%#015(Reading database ... 90%#015(Reading database ... 95%#015(Reading database ... 100%#015(Reading database ... 17714 files and directories currently installed.)
2023-07-04T09:20:49+02:00 Unpacking liblocale-gettext-perl (1.07-4) ...
2023-07-04T09:20:49+02:00 Preparing to unpack .../00-liblocale-gettext-perl_1.07-4_amd64.deb ...
2023-07-04T09:20:50+02:00 Unpacking ca-certificates (20230311ubuntu0.20.04.1) over (20211016ubuntu0.20.04.1) ...
2023-07-04T09:20:50+02:00 Unpacking readline-common (8.0-4) ...
2023-07-04T09:20:50+02:00 Preparing to unpack .../02-readline-common_8.0-4_all.deb ...
2023-07-04T09:20:50+02:00 Selecting previously unselected package readline-common.
2023-07-04T09:20:50+02:00 Unpacking xz-utils (5.2.4-1ubuntu1.1) ...
2023-07-04T09:20:50+02:00 Selecting previously unselected package libstdc++-9-dev:amd64.
2023-07-04T09:20:50+02:00 Preparing to unpack .../04-xz-utils_5.2.4-1ubuntu1.1_amd64.deb ...
2023-07-04T09:20:50+02:00 Preparing to unpack .../03-libreadline8_8.0-4_amd64.deb ...
2023-07-04T09:20:50+02:00 Selecting previously unselected package libreadline8:amd64.
2023-07-04T09:20:50+02:00 Selecting previously unselected package xz-utils.
2023-07-04T09:20:50+02:00 Unpacking libreadline8:amd64 (8.0-4) ...
2023-07-04T09:20:51+02:00 Selecting previously unselected package g++-9.
2023-07-04T09:20:51+02:00 Preparing to unpack .../05-libstdc++-9-dev_9.4.0-1ubuntu1~20.04.1_amd64.deb ...
2023-07-04T09:20:51+02:00 Unpacking libstdc++-9-dev:amd64 (9.4.0-1ubuntu1~20.04.1) ...
2023-07-04T09:20:51+02:00 Unpacking g++-9 (9.4.0-1ubuntu1~20.04.1) ...
2023-07-04T09:20:51+02:00 Preparing to unpack .../06-g++-9_9.4.0-1ubuntu1~20.04.1_amd64.deb ...
2023-07-04T09:20:52+02:00 Selecting previously unselected package g++.
2023-07-04T09:20:52+02:00 Unpacking libdpkg-perl (1.19.7ubuntu3.2) ...
2023-07-04T09:20:52+02:00 Unpacking make (4.2.1-1.2) ...
2023-07-04T09:20:52+02:00 Unpacking g++ (4:9.3.0-1ubuntu2) ...
2023-07-04T09:20:52+02:00 Preparing to unpack .../09-libdpkg-perl_1.19.7ubuntu3.2_all.deb ...
2023-07-04T09:20:52+02:00 Selecting previously unselected package make.
2023-07-04T09:20:52+02:00 Preparing to unpack .../07-g++_4%3a9.3.0-1ubuntu2_amd64.deb ...
2023-07-04T09:20:52+02:00 Selecting previously unselected package libdpkg-perl.
2023-07-04T09:20:52+02:00 Preparing to unpack .../08-make_4.2.1-1.2_amd64.deb ...
2023-07-04T09:20:52+02:00 Selecting previously unselected package dpkg-dev.
2023-07-04T09:20:52+02:00 Unpacking build-essential (12.8ubuntu1.1) ...
2023-07-04T09:20:52+02:00 Selecting previously unselected package libassuan0:amd64.
2023-07-04T09:20:52+02:00 Preparing to unpack .../11-build-essential_12.8ubuntu1.1_amd64.deb ...
2023-07-04T09:20:52+02:00 Unpacking libassuan0:amd64 (2.5.3-7ubuntu2) ...
2023-07-04T09:20:52+02:00 Preparing to unpack .../12-libassuan0_2.5.3-7ubuntu2_amd64.deb ...
2023-07-04T09:20:52+02:00 Selecting previously unselected package build-essential.
2023-07-04T09:20:52+02:00 Preparing to unpack .../10-dpkg-dev_1.19.7ubuntu3.2_all.deb ...
2023-07-04T09:20:52+02:00 Unpacking dpkg-dev (1.19.7ubuntu3.2) ...
2023-07-04T09:20:52+02:00 Selecting previously unselected package gpgconf.
2023-07-04T09:20:52+02:00 Preparing to unpack .../14-libksba8_1.3.5-2ubuntu0.20.04.2_amd64.deb ...
2023-07-04T09:20:52+02:00 Unpacking gpgconf (2.2.19-3ubuntu2.2) ...
2023-07-04T09:20:52+02:00 Selecting previously unselected package libksba8:amd64.
2023-07-04T09:20:52+02:00 Preparing to unpack .../13-gpgconf_2.2.19-3ubuntu2.2_amd64.deb ...
2023-07-04T09:20:52+02:00 Preparing to unpack .../15-libnpth0_1.6-1_amd64.deb ...
2023-07-04T09:20:52+02:00 Selecting previously unselected package libnpth0:amd64.
2023-07-04T09:20:52+02:00 Unpacking libnpth0:amd64 (1.6-1) ...
2023-07-04T09:20:52+02:00 Unpacking libksba8:amd64 (1.3.5-2ubuntu0.20.04.2) ...
2023-07-04T09:20:53+02:00 Selecting previously unselected package dirmngr.
2023-07-04T09:20:53+02:00 Preparing to unpack .../16-dirmngr_2.2.19-3ubuntu2.2_amd64.deb ...
2023-07-04T09:20:53+02:00 Unpacking dirmngr (2.2.19-3ubuntu2.2) ...
2023-07-04T09:20:53+02:00 Selecting previously unselected package libfakeroot:amd64.
2023-07-04T09:20:53+02:00 Selecting previously unselected package fakeroot.
2023-07-04T09:20:53+02:00 Unpacking libfakeroot:amd64 (1.24-1) ...
2023-07-04T09:20:53+02:00 Preparing to unpack .../17-libfakeroot_1.24-1_amd64.deb ...
2023-07-04T09:20:53+02:00 Preparing to unpack .../18-fakeroot_1.24-1_amd64.deb ...
2023-07-04T09:20:53+02:00 Unpacking gnupg-l10n (2.2.19-3ubuntu2.2) ...
2023-07-04T09:20:53+02:00 Selecting previously unselected package gnupg-l10n.
2023-07-04T09:20:53+02:00 Selecting previously unselected package gpg.
2023-07-04T09:20:53+02:00 Selecting previously unselected package gnupg-utils.
2023-07-04T09:20:53+02:00 Unpacking fakeroot (1.24-1) ...
2023-07-04T09:20:53+02:00 Preparing to unpack .../20-gnupg-utils_2.2.19-3ubuntu2.2_amd64.deb ...
2023-07-04T09:20:53+02:00 Preparing to unpack .../19-gnupg-l10n_2.2.19-3ubuntu2.2_all.deb ...
2023-07-04T09:20:53+02:00 Unpacking gnupg-utils (2.2.19-3ubuntu2.2) ...
2023-07-04T09:20:53+02:00 Unpacking gpg (2.2.19-3ubuntu2.2) ...
2023-07-04T09:20:53+02:00 Preparing to unpack .../21-gpg_2.2.19-3ubuntu2.2_amd64.deb ...
2023-07-04T09:20:53+02:00 Selecting previously unselected package gpg-agent.
2023-07-04T09:20:53+02:00 Selecting previously unselected package gpg-wks-client.
2023-07-04T09:20:53+02:00 Preparing to unpack .../23-gpg-agent_2.2.19-3ubuntu2.2_amd64.deb ...
2023-07-04T09:20:53+02:00 Selecting previously unselected package pinentry-curses.
2023-07-04T09:20:53+02:00 Preparing to unpack .../22-pinentry-curses_1.1.0-3build1_amd64.deb ...
2023-07-04T09:20:53+02:00 Unpacking pinentry-curses (1.1.0-3build1) ...
2023-07-04T09:20:53+02:00 Unpacking gpg-agent (2.2.19-3ubuntu2.2) ...
2023-07-04T09:20:54+02:00 Selecting previously unselected package gpgsm.
2023-07-04T09:20:54+02:00 Selecting previously unselected package gpg-wks-server.
2023-07-04T09:20:54+02:00 Unpacking gpgsm (2.2.19-3ubuntu2.2) ...
2023-07-04T09:20:54+02:00 Preparing to unpack .../25-gpg-wks-server_2.2.19-3ubuntu2.2_amd64.deb ...
2023-07-04T09:20:54+02:00 Unpacking gpg-wks-client (2.2.19-3ubuntu2.2) ...
2023-07-04T09:20:54+02:00 Preparing to unpack .../24-gpg-wks-client_2.2.19-3ubuntu2.2_amd64.deb ...
2023-07-04T09:20:54+02:00 Preparing to unpack .../26-gpgsm_2.2.19-3ubuntu2.2_amd64.deb ...
2023-07-04T09:20:54+02:00 Selecting previously unselected package gnupg.
2023-07-04T09:20:54+02:00 Unpacking gpg-wks-server (2.2.19-3ubuntu2.2) ...
2023-07-04T09:20:54+02:00 Preparing to unpack .../27-gnupg_2.2.19-3ubuntu2.2_all.deb ...
2023-07-04T09:20:54+02:00 Selecting previously unselected package libalgorithm-merge-perl.
2023-07-04T09:20:54+02:00 Selecting previously unselected package libalgorithm-diff-xs-perl.
2023-07-04T09:20:54+02:00 Selecting previously unselected package libalgorithm-diff-perl.
2023-07-04T09:20:54+02:00 Preparing to unpack .../29-libalgorithm-diff-xs-perl_0.04-6_amd64.deb ...
2023-07-04T09:20:54+02:00 Preparing to unpack .../28-libalgorithm-diff-perl_1.19.03-2_all.deb ...
2023-07-04T09:20:54+02:00 Unpacking libalgorithm-diff-perl (1.19.03-2) ...
2023-07-04T09:20:54+02:00 Unpacking libalgorithm-diff-xs-perl (0.04-6) ...
2023-07-04T09:20:54+02:00 Unpacking gnupg (2.2.19-3ubuntu2.2) ...
2023-07-04T09:20:54+02:00 Unpacking libalgorithm-merge-perl (0.08-3) ...
2023-07-04T09:20:54+02:00 Preparing to unpack .../30-libalgorithm-merge-perl_0.08-3_all.deb ...
2023-07-04T09:20:54+02:00 Preparing to unpack .../31-libfile-fcntllock-perl_0.22-3build4_amd64.deb ...
2023-07-04T09:20:54+02:00 Setting up libksba8:amd64 (1.3.5-2ubuntu0.20.04.2) ...
2023-07-04T09:20:54+02:00 Selecting previously unselected package libfile-fcntllock-perl.
2023-07-04T09:20:54+02:00 Unpacking libfile-fcntllock-perl (0.22-3build4) ...
2023-07-04T09:20:54+02:00 Setting up libstdc++-9-dev:amd64 (9.4.0-1ubuntu1~20.04.1) ...
2023-07-04T09:20:54+02:00 Setting up libalgorithm-diff-perl (1.19.03-2) ...
2023-07-04T09:20:54+02:00 Setting up libfile-fcntllock-perl (0.22-3build4) ...
2023-07-04T09:20:54+02:00 Setting up libnpth0:amd64 (1.6-1) ...
2023-07-04T09:20:54+02:00 Setting up fakeroot (1.24-1) ...
2023-07-04T09:20:54+02:00 update-alternatives: warning: skip creation of /usr/share/man/man1/faked.1.gz because associated file /usr/share/man/man1/faked-sysv.1.gz (of link group fakeroot) doesn't exist
2023-07-04T09:20:54+02:00 update-alternatives: warning: skip creation of /usr/share/man/man1/fakeroot.1.gz because associated file /usr/share/man/man1/fakeroot-sysv.1.gz (of link group fakeroot) doesn't exist
2023-07-04T09:20:54+02:00 Setting up libassuan0:amd64 (2.5.3-7ubuntu2) ...
2023-07-04T09:20:54+02:00 update-alternatives: warning: skip creation of /usr/share/man/es/man1/fakeroot.1.gz because associated file /usr/share/man/es/man1/fakeroot-sysv.1.gz (of link group fakeroot) doesn't exist
2023-07-04T09:20:54+02:00 update-alternatives: using /usr/bin/fakeroot-sysv to provide /usr/bin/fakeroot (fakeroot) in auto mode
2023-07-04T09:20:54+02:00 Setting up libfakeroot:amd64 (1.24-1) ...
2023-07-04T09:20:54+02:00 update-alternatives: warning: skip creation of /usr/share/man/fr/man1/fakeroot.1.gz because associated file /usr/share/man/fr/man1/fakeroot-sysv.1.gz (of link group fakeroot) doesn't exist
2023-07-04T09:20:54+02:00 update-alternatives: warning: skip creation of /usr/share/man/es/man1/faked.1.gz because associated file /usr/share/man/es/man1/faked-sysv.1.gz (of link group fakeroot) doesn't exist
2023-07-04T09:20:54+02:00 Setting up ca-certificates (20230311ubuntu0.20.04.1) ...
2023-07-04T09:20:54+02:00 update-alternatives: warning: skip creation of /usr/share/man/sv/man1/faked.1.gz because associated file /usr/share/man/sv/man1/faked-sysv.1.gz (of link group fakeroot) doesn't exist
2023-07-04T09:20:54+02:00 update-alternatives: warning: skip creation of /usr/share/man/sv/man1/fakeroot.1.gz because associated file /usr/share/man/sv/man1/fakeroot-sysv.1.gz (of link group fakeroot) doesn't exist
2023-07-04T09:20:54+02:00 update-alternatives: warning: skip creation of /usr/share/man/fr/man1/faked.1.gz because associated file /usr/share/man/fr/man1/faked-sysv.1.gz (of link group fakeroot) doesn't exist
2023-07-04T09:20:55+02:00 debconf: unable to initialize frontend: Dialog
2023-07-04T09:20:55+02:00 debconf: (TERM is not set, so the dialog frontend is not usable.)
2023-07-04T09:20:55+02:00 debconf: falling back to frontend: Readline
2023-07-04T09:20:58+02:00 Updating certificates in /etc/ssl/certs...
2023-07-04T09:21:01+02:00 rehash: warning: skipping ca-certificates.crt,it does not contain exactly one certificate or CRL
2023-07-04T09:21:01+02:00 Setting up xz-utils (5.2.4-1ubuntu1.1) ...
2023-07-04T09:21:01+02:00 Setting up gnupg-l10n (2.2.19-3ubuntu2.2) ...
2023-07-04T09:21:01+02:00 Setting up make (4.2.1-1.2) ...
2023-07-04T09:21:01+02:00 19 added, 6 removed; done.
2023-07-04T09:21:01+02:00 update-alternatives: warning: skip creation of /usr/share/man/man1/unlzma.1.gz because associated file /usr/share/man/man1/unxz.1.gz (of link group lzma) doesn't exist
2023-07-04T09:21:01+02:00 update-alternatives: warning: skip creation of /usr/share/man/man1/lzless.1.gz because associated file /usr/share/man/man1/xzless.1.gz (of link group lzma) doesn't exist
2023-07-04T09:21:01+02:00 update-alternatives: warning: skip creation of /usr/share/man/man1/lzcat.1.gz because associated file /usr/share/man/man1/xzcat.1.gz (of link group lzma) doesn't exist
2023-07-04T09:21:01+02:00 update-alternatives: warning: skip creation of /usr/share/man/man1/lzmore.1.gz because associated file /usr/share/man/man1/xzmore.1.gz (of link group lzma) doesn't exist
2023-07-04T09:21:01+02:00 update-alternatives: warning: skip creation of /usr/share/man/man1/lzma.1.gz because associated file /usr/share/man/man1/xz.1.gz (of link group lzma) doesn't exist
2023-07-04T09:21:01+02:00 update-alternatives: using /usr/bin/xz to provide /usr/bin/lzma (lzma) in auto mode
2023-07-04T09:21:01+02:00 update-alternatives: warning: skip creation of /usr/share/man/man1/lzdiff.1.gz because associated file /usr/share/man/man1/xzdiff.1.gz (of link group lzma) doesn't exist
2023-07-04T09:21:01+02:00 update-alternatives: warning: skip creation of /usr/share/man/man1/lzegrep.1.gz because associated file /usr/share/man/man1/xzegrep.1.gz (of link group lzma) doesn't exist
2023-07-04T09:21:01+02:00 update-alternatives: warning: skip creation of /usr/share/man/man1/lzcmp.1.gz because associated file /usr/share/man/man1/xzcmp.1.gz (of link group lzma) doesn't exist
2023-07-04T09:21:01+02:00 Setting up g++-9 (9.4.0-1ubuntu1~20.04.1) ...
2023-07-04T09:21:01+02:00 update-alternatives: warning: skip creation of /usr/share/man/man1/lzfgrep.1.gz because associated file /usr/share/man/man1/xzfgrep.1.gz (of link group lzma) doesn't exist
2023-07-04T09:21:01+02:00 update-alternatives: warning: skip creation of /usr/share/man/man1/lzgrep.1.gz because associated file /usr/share/man/man1/xzgrep.1.gz (of link group lzma) doesn't exist
2023-07-04T09:21:01+02:00 Setting up g++ (4:9.3.0-1ubuntu2) ...
2023-07-04T09:21:01+02:00 Setting up libdpkg-perl (1.19.7ubuntu3.2) ...
2023-07-04T09:21:01+02:00 update-alternatives: warning: skip creation of /usr/share/man/man1/c++.1.gz because associated file /usr/share/man/man1/g++.1.gz (of link group c++) doesn't exist
2023-07-04T09:21:01+02:00 update-alternatives: using /usr/bin/g++ to provide /usr/bin/c++ (c++) in auto mode
2023-07-04T09:21:01+02:00 Setting up liblocale-gettext-perl (1.07-4) ...
2023-07-04T09:21:01+02:00 Setting up readline-common (8.0-4) ...
2023-07-04T09:21:01+02:00 Setting up libalgorithm-diff-xs-perl (0.04-6) ...
2023-07-04T09:21:01+02:00 Setting up libalgorithm-merge-perl (0.08-3) ...
2023-07-04T09:21:01+02:00 Setting up dpkg-dev (1.19.7ubuntu3.2) ...
2023-07-04T09:21:01+02:00 Setting up libreadline8:amd64 (8.0-4) ...
2023-07-04T09:21:01+02:00 Setting up gnupg-utils (2.2.19-3ubuntu2.2) ...
2023-07-04T09:21:01+02:00 Setting up build-essential (12.8ubuntu1.1) ...
2023-07-04T09:21:01+02:00 Setting up pinentry-curses (1.1.0-3build1) ...
2023-07-04T09:21:01+02:00 Setting up gpg (2.2.19-3ubuntu2.2) ...
2023-07-04T09:21:01+02:00 Setting up gpgconf (2.2.19-3ubuntu2.2) ...
2023-07-04T09:21:01+02:00 Setting up gpg-agent (2.2.19-3ubuntu2.2) ...
2023-07-04T09:21:02+02:00 Setting up gpgsm (2.2.19-3ubuntu2.2) ...
2023-07-04T09:21:02+02:00 Setting up dirmngr (2.2.19-3ubuntu2.2) ...
2023-07-04T09:21:02+02:00 Setting up gpg-wks-server (2.2.19-3ubuntu2.2) ...
2023-07-04T09:21:02+02:00 Setting up gpg-wks-client (2.2.19-3ubuntu2.2) ...
2023-07-04T09:21:02+02:00 Setting up gnupg (2.2.19-3ubuntu2.2) ...
2023-07-04T09:21:02+02:00 Updating certificates in /etc/ssl/certs...
2023-07-04T09:21:02+02:00 Processing triggers for ca-certificates (20230311ubuntu0.20.04.1) ...
2023-07-04T09:21:02+02:00 Processing triggers for libc-bin (2.31-0ubuntu9.9) ...
2023-07-04T09:21:02+02:00 #015
2023-07-04T09:21:02+02:00 /sbin/ldconfig.real: /lib/libproj.so.22 is not a symbolic link
2023-07-04T09:21:04+02:00 Running hooks in /etc/ca-certificates/update.d...
2023-07-04T09:21:04+02:00 0 added, 0 removed; done.
2023-07-04T09:21:04+02:00 #015
2023-07-04T09:21:05+02:00 done.
2023-07-04T09:21:05+02:00 done.
2023-07-04T09:21:05+02:00 time="2023-07-04T07:21:04.912633009Z" level=warning msg="cleaning up after shim disconnected" id=44f799d3020ecb492b566ee1f965f56f178294398c7988e606370cd6df82cc65 namespace=moby
2023-07-04T09:21:05+02:00 time="2023-07-04T07:21:04.911960433Z" level=info msg="shim disconnected" id=44f799d3020ecb492b566ee1f965f56f178294398c7988e606370cd6df82cc65 namespace=moby
2023-07-04T09:21:05+02:00 time="2023-07-04T07:21:04.912692429Z" level=info msg="cleaning up dead shim" namespace=moby
2023-07-04T09:21:05+02:00 time="2023-07-04T07:21:04.912043965Z" level=info msg="ignoring event" container=44f799d3020ecb492b566ee1f965f56f178294398c7988e606370cd6df82cc65 module=libcontainerd namespace=moby topic=/tasks/delete type="*events.TaskDelete"
2023-07-04T09:21:08+02:00 ---> Running in 4db7157560ce
2023-07-04T09:21:08+02:00 ---> 22494725f645
2023-07-04T09:21:08+02:00 time="2023-07-04T07:21:08.994770909Z" level=info msg="loading plugin \"io.containerd.ttrpc.v1.pause\"..." runtime=io.containerd.runc.v2 type=io.containerd.ttrpc.v1
2023-07-04T09:21:08+02:00 Removing intermediate container 44f799d3020e
2023-07-04T09:21:08+02:00 time="2023-07-04T07:21:08.994842358Z" level=info msg="loading plugin \"io.containerd.ttrpc.v1.task\"..." runtime=io.containerd.runc.v2 type=io.containerd.ttrpc.v1
2023-07-04T09:21:08+02:00 time="2023-07-04T07:21:08.994810512Z" level=info msg="loading plugin \"io.containerd.event.v1.publisher\"..." runtime=io.containerd.runc.v2 type=io.containerd.event.v1
```

</details>